MobileCenterDistribute/MobileCenterDistribute/MSDistributePrivate.h

```diff
@@ -43,6 +43,11 @@ static NSString *const kMSUpdateTokenKey = @"MSUpdateToken";
 @interface MSDistribute ()
 
 /**
+ * View controller presenting the `SFSafariViewController`.
+ */
+@property(nonatomic) UIViewController *safariHostingViewController;
```

Comment:
We have to automatically dispose it later for instance if the module is disabled or if the openURL method is called with the right token. To do so we need to access the hosting view controller hence the property :)